ultravox/model/ultravox_model.py

```diff
@@ -286,13 +287,16 @@ def _create_audio_tower(
                 audio_tower = ModifiedWhisperEncoder.from_pretrained(
                     config.audio_model_id, torch_dtype=config.torch_dtype
                 )
+                audio_tower.init_latency_mask(
+                    config.audio_latency_block_size, dtype=config.torch_dtype)
```

Comment:
Thanks. It would be good to have an assert in the other case as well if block_size is non-zero rather than silently refusing to use it.